simulator/src/main/java/com/hazelcast/simulator/worker/MemberWorker.java

```diff
@@ -69,8 +69,6 @@
 
     MemberWorker(WorkerType type, String publicAddress, int agentIndex, int workerIndex, int workerPort, String hzConfigFile,
                  boolean autoCreateHzInstance, int workerPerformanceMonitorIntervalSeconds) throws Exception {
-        SHUTDOWN_STARTED.set(false);
```

Comment:
Can you please restore this line? There are test failures without it and it has nothing to do with the `WorkerPerformanceMonitor`. Can you also please run all tests on your branch? This just takes 2.5 minutes for Simulator.